lib/utils/index.js

```diff
@@ -341,17 +341,37 @@ function hasChangesInForbiddenSections(diff) {
     .map(line => _.nth(/^([a-z]\w+):/.exec(line), 1))
     .difference([
       'update',
-      'compilation',
       'releases',
-      'resource_pools',
-      'networks',
-      'properties',
       'tags',
       'addons'
     ])
     .value();
+
+  function isJobRemoved() {
+    const jobsRegex = new RegExp('^  jobs'); // this regex is to find the position of jobs section
+    const jobsLevelRegex = new RegExp('^  [a-z]+'); // this regex is to find the position of next section at the same level as jobs
+    const jobNameRegex = new RegExp('^  [- ] name'); // this regex is to find the position of removal in job name
+    const jobStartIndex = _.findIndex(diff, element => {
+      return jobsRegex.test(element[0]);
+    });
+    const jobEndIndex = _.findIndex(diff, element => {
+      return jobsLevelRegex.test(element[0]);
+    }, jobStartIndex + 1);
+
+    const jobDiff = _.slice(diff, jobStartIndex, jobEndIndex);
+    const jobNameRemovedIndex = _.findIndex(jobDiff, element => {
+      return jobNameRegex.test(element[0]) && _.includes(element[1], 'removed');
+    });
+    if (jobNameRemovedIndex !== -1) {
+      return true;
+    } // this checks if the name removal is present within the jobs section only.
+    return false;
+  }
   if (!_.isEmpty(forbiddenSections) && !_.includes(forbiddenSections, 'director_uuid')) {
-    throw new errors.Forbidden(`Automatic update not possible. Detected changes in forbidden section(s) '${forbiddenSections.join(',')}'`);
+    const forbiddenSectionsDiff = _.filter(diff, element => _.includes(element[0], 'instances') || _.includes(element[0], 'persistent_disk_type'));
+    if (!_.isEmpty(forbiddenSectionsDiff) || isJobRemoved()) {
+      throw new errors.Forbidden(`Automatic update not possible. Detected changes in forbidden section(s) '${forbiddenSections.join(',')}'`);
```

Comment:
change the error message here to reflect the actual changes. for ex, it should reflect the values from `forbiddenSectionsDiff` and not from `forbiddenSections`

Reply:
ok.. I will change the message